protos/apigw/v1/apigw.proto

```diff
@@ -36,6 +60,20 @@ message Service {
   string docs_path = 2;
 }
 
+message TerraformEntity {
+  TerraformEntityMethodType type = 1;
+  string name = 2;
+}
+
+enum TerraformEntityMethodType {
```

Comment:
can this be an embeded enum on the TerraformEntity message? do you see us re-using it outside this message?

Reply:
changed in latest commit